website/pages/docs/nia/architecture.mdx

```diff
@@ -6,4 +6,38 @@ description: >-
   Consul-Terraform-Sync Architecture
 ---
 
-# Consul-Terraform-Sync Architecture
+# Consul Terraform Sync Architecture
+
+Consul Terraform Sync is a service-oriented tool for managing
+network infrastructure near real-time. Consul-Terraform-Sync runs as a daemon
+and integrates the network topology maintained by your Consul cluster with your
+network infrastructure to dynamically secure and connect services.
+
+## 10,000 Foot View
+
+[![Consul Terraform Sync Architecture](/img/nia-highlevel-diagram.png)](/img/nia-highlevel-diagram.png)
+
+The diagram shows Consul Terraform Sync monitoring the Consul service catalog
```

Comment:
minor thing, I think "Consul Terraform Sync" => "Consul-Terraform-Sync". Unless we want to norm the other way around or there's different use case for the two?

Reply:
I wasn't sure about this. I initially had the dashes in there, but then saw it in another doc without. The dashes seemed like they might be an artifact of the command line/repo name where spaces are bad. With this being a name/title I wasn't sure if we should keep them.

Anyways. I'll add them back as re-looking at other docs I see we are using them everywhere now and I don't remember where I even saw it without (ie. not sure if it was changed or I was looking at something old).

Reply:
Yea, there's definitely a mix of with or without dashes. From what I remember, the repo has without dashes and the consul.io docs has with dashes. 🤔 we could discuss and pick one to stick with